python/cucim/src/cucim/skimage/filters/tests/test_median.py

```diff
@@ -249,3 +250,17 @@ def test_median_preserve_dtype(image, dtype):
 )
 def test_median(img, behavior):
     median(img, behavior=behavior)
+
+
+def test_median_nonsquare():
```

Comment:
Not sure why this shows up here in the diff. The changes in this file are from:
https://github.com/rapidsai/cucim/pull/521